### PR TITLE
feat(core): ARC-1906 fetch env from ssm

### PR DESCRIPTION
**Pull request by sfvishalgupta**
Added code to support ssm values in Lambda Environment variables.

### CODE REVIEW

File: src/aws/lambda/utils.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import * as aws from '@cdktf/provider-aws';
+import { Construct } from "constructs";
+import { IEnvironmentVars } from './interface'
+
+// import { resolve } from 'path';
```
Comment (jamescrowley321) [inline src/aws/lambda/utils.ts:5]: Is sonar running on this repo? Something like this should be picked up there or in linting rules during the pre commit.

**Review by jamescrowley321 — CHANGES_REQUESTED** [review]

File: src/aws/lambda/utils.ts (hunk 1 above)
Comment (sfvishalgupta) [reply src/aws/lambda/utils.ts]: Hi @jamescrowley321 
I dont think sonar is there in this repo although i have remove all the commented code, any thing else needs to be change in above?

Comment (jamescrowley321) [reply src/aws/lambda/utils.ts]: Let's try it out.

Comment (jamescrowley321) [reply src/aws/lambda/utils.ts]: Update the README and add a few examples we can use to test and then we will merge.

**Review by jamescrowley321 — APPROVED** [review]

**Pull request merged**